youtube_dl/extractor/rtvslo.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    unified_timestamp,
+    try_get, ExtractorError
+)
+
+
+class RTVSLO4DIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:4d|www)\.rtvslo\.si/(?:arhiv/.+|embed|4d/arhiv)/(?P<id>\d+)'
```

Comment:
accept only valid URLs, add `only_matching` tests.

Reply:
Are there any problematic URLs? Looks like it matches exactly what it should: https://regex101.com/r/tTMFCg/1 (see unit tests tab on the left).

I've added `only_matching` tests, but can't find a way to run them. Is there one?

Reply:
you should try not to match invalid URLs(ex: https://rtvslo.si/embed/174595438, https://4d.rtvslo.si/4d/arhiv/seje-odbora-za-kmetijstvo-gozdarstvo-in-prehrano/174595438, ...).